client/tests/unit/new-search.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { createLocalVue, mount, shallowMount } from '@vue/test-utils'
+import NewSearch from '@/components/new-request/search.vue'
+import newReqModule from '@/store/new-request-module'
+import Vuetify from 'vuetify'
+
+const localVue = createLocalVue()
+const vuetify = new Vuetify()
+
+localVue.use(Vuetify)
+
+describe('name-input.vue', () => {
+  let wrapper: any
+
+  beforeEach(async () => {
+    wrapper = mount(NewSearch, {
+      localVue,
+      vuetify
+    })
+    newReqModule.mutateEntityType('CR')
+    newReqModule.mutateLocation('BC')
+    newReqModule.mutateRequestAction('NEW')
+    newReqModule.mutateName('test valid name')
+    await wrapper.vm.$nextTick()
+    wrapper.vm.clearErrors()
+  })
+
+  it('Resists submission when there is no action selected', async () => {
```

Comment:
"Resists" LOL! (It's totally fine though.)

Reply:
haha there's a test for the name input component that is similar and it uses this wording lol